modules/GanneffServ.rb

```diff
@@ -326,7 +391,7 @@ def BADSERV(client, parv = [])
       debug(LOG_DEBUG, "#{client.name} called BADSERV and the parms are #{parv.join(",")}")
       server = parv[1].downcase
 
-      if server =~ /.*\.oftc.net$/
+      if server =~ /.*\.oftc\.net$/
```

Comment:
This is unrelated, but included because I was thinking about regular expressions while working on this file. It's an outlier in this file, everything else included the `\`.